frontend/public/components/monitoring/receiver-forms/slack-receiver-form.tsx

```diff
@@ -72,6 +85,144 @@ export const Form: React.FC<FormProps> = ({ globals, formValues, dispatchFormCha
           The Slack channel or user to send notifications to
         </div>
       </div>
+      <div className="form-group">
+        <ExpandCollapse
+          textCollapsed="Show advanced configuration"
+          textExpanded="Hide advanced configuration"
+        >
+          <div className="co-form-subsection">
+            <div className="form-group">
+              <SendResolvedAlertsCheckbox
+                formField="slack_send_resolved"
+                formValues={formValues}
+                dispatchFormChange={dispatchFormChange}
+              />
+            </div>
+            <div className="form-group">
+              <label className="control-label" htmlFor="slack-icon-type">
+                Icon &nbsp;
+                <RadioInput
+                  title="Url"
+                  name="slackIconType"
+                  id="slack-icon-type"
+                  value="url"
+                  onChange={(e) =>
+                    dispatchFormChange({
+                      type: 'setFormValues',
+                      payload: { slackIconType: e.target.value },
+                    })
+                  }
+                  checked={formValues.slackIconType === 'url'}
+                  inline
+                />
+                <RadioInput
+                  title="Emoji"
+                  name="slackIconType"
+                  value="emoji"
+                  data-test-id="slack-icon-type-emoji"
+                  onChange={(e) =>
+                    dispatchFormChange({
+                      type: 'setFormValues',
+                      payload: { slackIconType: e.target.value },
+                    })
+                  }
+                  checked={formValues.slackIconType === 'emoji'}
+                  inline
+                />
+              </label>
+              {formValues.slackIconType === 'url' && (
+                <>
+                  <input
+                    className="pf-c-form-control"
+                    type="text"
+                    aria-describedby="slack-icon_url-help"
+                    data-test-id="slack-icon_url"
+                    value={formValues.slack_icon_url}
+                    onChange={(e) =>
+                      dispatchFormChange({
+                        type: 'setFormValues',
+                        payload: { slack_icon_url: e.target.value },
+                      })
+                    }
+                  />
+                  <div className="help-block" id="slack-icon_url-help">
+                    The url of the icon
```

Comment:
All of the help blocks should have punctuation to be consistent with other forms in console (although it looks like existing help blocks for Alertmanager is missing that).

```suggestion
                    The URL of the icon.
```